make-prs.py

```diff
@@ -0,0 +1,251 @@
+import argparse
```

Comment:
It would be nice if there was a module-level docstring that briefly explained what the file does (it could even say to read the HACKING.md file for more details).